spec/controllers/user_controller_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+require 'rails-controller-testing'
+Rails::Controller::Testing.install
+
+RSpec.describe UsersController, type: :request do
+  describe 'GET #index' do
+    it 'returns a successful response' do
+      get users_path
+      expect(response).to be_successful
+      expect(response).to have_http_status(200)
+    end
+
+    it 'renders the correct template' do
+      get users_path
+      expect(response).to render_template(:index)
+    end
+
+    it 'includes the correct placeholder text' do
+      get users_path
+      expect(response.body).to include('List of users')
+    end
+  end
```

Comment:
- [ ] `README.md` file: I think it would be better if you fix the broken `[MIT](./LICENSE)` link in your README file to make it more professional as it shows a **404** error page once clicked. And to fix that, you can only create a new file name `LICENSE` in your root directory and get its content from this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)** or rename this `[MIT](./LICENSE)` to `[MIT](./MIT.md)`, then the created file in your directory name it `MIT.md` and use the same content in this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)**

- [ **OPTIONAL** ] `README.md` file: What do you think if you update your current `Live Demo` link with the working one as it shows a blank page? or if you don't have the working link you can leave there a message similar to these: `N/A`, `Available soon`,... This will make your README looks more professional